scripts/events_module/patrol/patrol_outcome.py

```diff
@@ -491,19 +492,28 @@ def _handle_death(self, patrol: "Patrol") -> str:
                 elif "some_lives" in self.dead_cats:
                     lives_lost = random.randint(1, max(1, game.clan.leader_lives - 1))
                     game.clan.leader_lives -= lives_lost
-                    results.append(
+                    if lives_lost > 1:
+                        results.append(
                         event_text_adjust(
                             Cat,
-                            i18n.t("cat.history.leader_death_all", count=lives_lost),
+                            i18n.t("leader_lost_lives.many") % {"count": lives_lost},
```

Comment:
Have you tested this? I'm not familiar with this syntax, and I don't think it's correctly pointing to the language string.

Also, this entire if-else block is not necessary because i18n should be handling pluralization. That's what the `one` and `many` labels are doing, look on the wiki for more information.

Here's what that line should look like based on my understanding.
```suggestion
                            i18n.t("cat.history.leader_lost_lives", count=lives_lost),
```